### DISCUSSION TITLE
Repetitive Invocation of a single action

### DISCUSSION THREAD

**Discussion by mrprigun** [moleculerjs/moleculer#1235] [Q&A]
Hello, I encountered certain obstacles in my use case while attempting to execute the molecular actions. Briefly, I have a service (gateway for the main app with `moleculer-web`)  that looks like this :

```js
module.exports = {
  name: "test",
  actions: {
    hello: {
      rest: "GET /hello",
      handler(ctx) {
        ctx.meta.$responseType = "text/plain";
        return "Hello Moleculer";
      }
    },

    second_call: {
      rest: "GET /second_call",
      handler(ctx) {
        // some actions
        return ctx.call('external.call', params_action_results_above);
      }
    },
  }
};
```

`external.call` is located on another node and invocation happens using NATS transporter. By design actions `hello` and `second_call` are gonna be loaded to process unique tasks. Before publishing it to production I made benchmarks for each action and received such results on my laptop:

- The `hello` action works pretty well, the result was ~600rps 
- But the second one `second_call` was very much degraded, I received just ~40rps

Also, I received similar results after deployment to a prod-like environment. I believe this occurs due to external triggering, which is anticipated, but why so much? I've tried to use different load-balancing strategies and `bulkhead` but didn't receive any significant improvements. Is there a possibility of configuring Moleculer to enhance this behavior or it's some kind of bug?

**Comment by icebob** [comment]
Issue opened: https://github.com/moleculerjs/moleculer/issues/1237
